lib/internal/error_serdes.js

```diff
@@ -48,13 +56,17 @@ function TryGetAllProperties(object, target = object) {
     if (getter && key !== '__proto__') {
       try {
         descriptor.value = FunctionPrototypeCall(getter, target);
+        delete descriptor.get;
+        delete descriptor.set;
       } catch {
         // Continue regardless of error.
       }
     }
-    if ('value' in descriptor && typeof descriptor.value !== 'function') {
-      delete descriptor.get;
-      delete descriptor.set;
+    if (key === 'cause') {
+      descriptor.value = serializeError(descriptor.value);
+      all[key] = descriptor;
+    } else if ('value' in descriptor &&
+            typeof descriptor.value !== 'function' && typeof descriptor.value !== 'symbol') {
       all[key] = descriptor;
```

Comment:
I realize that was there before this PR, so feel free to ignore it, but we should probably not pass the descriptor that inherits from `Object.prototype`, and instead re-create a null-prototype object with only the properties we want from it. Also we are accessing `descriptor.get` and `descriptor.value` without checking first if those are own properties.
If we're not addressing that in that PR, could we add a TODO?